### ISSUE TITLE
[V2]Ubuntu - wails doctor cannot check the user-installed package

### ISSUE THREAD

**Issue by CoderfishGzh** [wailsapp/wails#1327]
### Description

I am running Ubuntu.When running `wails doctor` to ensure i have all dependencies, but it report that i don't have `libgtk-3` and `libwebkit`installed, the status for them is "Available", but i have installed

### To Reproduce

1. Use `sudo apt install libgtk-3-dev` and `sudo apt install libwebkit2gtk-4.0-dev` to install  `libgtk-3` and `libwebkit`
2. run `wails doctor`
3. The output shows  `libgtk-3` and `libwebkit` statue is "Available" when in fact it should be "installed"

### Expected behaviour

`wails doctor` would report `libgtk-3` and `libwebkit` as installed 

### Screenshots

![安装依赖.png](https://s2.loli.net/2022/04/12/lpAf52kBucQRH8w.png)
![wails doctor.png](https://s2.loli.net/2022/04/12/fWTEqsih2lBo6zC.png)

### System Details

```shell
>wails doctor
Wails CLI v2.0.0-beta.34


Scanning system - Please wait (this may take a long time)...Done.

System
------
OS:			Ubuntu
Version: 		20.04
ID:			ubuntu
Go Version:		go1.18
Platform:		linux
Architecture:		amd64
Package Manager: 	apt

Dependency	Package Name		Status		Version
----------	------------		------		-------
*docker		docker.io		Available	20.10.7-0ubuntu5~20.04.2
gcc		build-essential		Installed	9.4.0
libgtk-3	libgtk-3-dev		Available	3.24.20-0ubuntu1.1
libwebkit	libwebkit2gtk-4.0-dev	Available	2.34.6-0ubuntu0.20.04.1
npm		npm			Installed	8.5.0
*nsis		nsis			Available	3.05-2
pkg-config	pkg-config		Installed	0.29.1

* - Optional Dependency

Diagnosis
---------
Your system has missing dependencies!

Required package(s) installation details: 
  - libgtk-3: sudo apt install libgtk-3-dev
  - libwebkit: sudo apt install libwebkit2gtk-4.0-dev

Optional package(s) installation details: 
  - docker: sudo apt install docker.io
  - nsis: sudo apt install nsis
```
```


### Additional context

_No response_

**Comment by leaanthony** [comment]
Hi. What's your system LOCALE?

**Comment by CoderfishGzh** [comment]
> 
You mean the command "LOCALE"?
```
oem@gzhlinux:~$ locale
LANG=zh_CN.UTF-8
LANGUAGE=zh_CN:zh
LC_CTYPE="zh_CN.UTF-8"
LC_NUMERIC="zh_CN.UTF-8"
LC_TIME="zh_CN.UTF-8"
LC_COLLATE="zh_CN.UTF-8"
LC_MONETARY="zh_CN.UTF-8"
LC_MESSAGES="zh_CN.UTF-8"
LC_PAPER="zh_CN.UTF-8"
LC_NAME="zh_CN.UTF-8"
LC_ADDRESS="zh_CN.UTF-8"
LC_TELEPHONE="zh_CN.UTF-8"
LC_MEASUREMENT="zh_CN.UTF-8"
LC_IDENTIFICATION="zh_CN.UTF-8"
LC_ALL=
```

**Comment by leaanthony** [comment]
@ianmjones any insight into this issue? Perhaps there's a better Linuxy way?

**Comment by ianmjones** [comment]
@leaanthony I just installed Go on a relatively clean Ubuntu 20.04 (Focal), cloned the wails repo, installed wails v2, and then ran `wails doctor`. It correctly noticed I didn't have `build-essentials`, `libgtk-3` and `libwebkit` installed yet, and gave the correct suggestions for how to install them.

So I did:

`sudo apt install build-essential libgtk-3-dev libwebkit2gtk-4.0-dev`

And now `wails` correctly sees them installed (I haven't installed `npm` yet):

```
ian@ubuntu-focal-qemu:~$ wails doctor
Wails CLI v2.0.0-beta.38

Scanning system - Please wait (this may take a long time)...Done.

System
------
OS:		Ubuntu
Version: 	20.04
ID:		ubuntu
Go Version:	go1.18.3
Platform:	linux
Architecture:	amd64

Wails
------
Version: 		v2.0.0-beta.38
Revision:		1cbc34eff8682e4daec25dd3bddb5bbc73ec09d5
Modified:		false
Package Manager: 	apt

Dependency	Package Name		Status		Version
----------	------------		------		-------
*docker		docker.io		Available	20.10.12-0ubuntu2~20.04.1
gcc		build-essential		Installed	12.8ubuntu1.1
libgtk-3	libgtk-3-dev		Installed	3.24.20-0ubuntu1.1
libwebkit	libwebkit2gtk-4.0-dev	Installed	2.36.3-0ubuntu0.20.04.1
npm		npm			Available	6.14.4+ds-1ubuntu2
*nsis		nsis			Available	3.05-2
pkg-config	pkg-config		Installed	0.29.1

* - Optional Dependency

Diagnosis
---------
Your system has missing dependencies!

Required package(s) installation details: 
  - npm: sudo apt install npm

Optional package(s) installation details: 
  - docker: sudo apt install docker.io
  - nsis: sudo apt install nsis
```

And manually running the code you pointed me to (thanks for that, very helpful), sure enough shows the expected output that contains the searched string "[installed]":

```
ian@ubuntu-focal-qemu:~$ apt list -qq libwebkit2gtk-4.0-dev
libwebkit2gtk-4.0-dev/focal-updates,focal-security,now 2.36.3-0ubuntu0.20.04.1 amd64 [installed]
libwebkit2gtk-4.0-dev/focal-updates,focal-security 2.36.3-0ubuntu0.20.04.1 i386
ian@ubuntu-focal-qemu:~$ apt list -qq libgtk-3-dev
libgtk-3-dev/focal-updates,now 3.24.20-0ubuntu1.1 amd64 [installed]
libgtk-3-dev/focal-updates 3.24.20-0ubuntu1.1 i386
```

So it's all fine for English language. Then I installed Chinese (simplified) language and switched to it, and confirmed the issue:

```
ian@ubuntu-focal-qemu:~$ apt list -qq libgtk-3-dev
libgtk-3-dev/focal-updates,now 3.24.20-0ubuntu1.1 amd64 [已安装]
libgtk-3-dev/focal-updates 3.24.20-0ubuntu1.1 i386
ian@ubuntu-focal-qemu:~$ wails doctor
Wails CLI v2.0.0-beta.38

Scanning system - Please wait (this may take a long time)...Done.

System
------
OS:		Ubuntu
Version: 	20.04
ID:		ubuntu
Go Version:	go1.18.3
Platform:	linux
Architecture:	amd64

Wails
------
Version: 		v2.0.0-beta.38
Revision:		1cbc34eff8682e4daec25dd3bddb5bbc73ec09d5
Modified:		false
Package Manager: 	apt

Dependency	Package Name		Status		Version
----------	------------		------		-------
*docker		docker.io		Available	20.10.12-0ubuntu2~20.04.1
gcc		build-essential		Installed	9.4.0
libgtk-3	libgtk-3-dev		Available	3.24.20-0ubuntu1.1
libwebkit	libwebkit2gtk-4.0-dev	Available	2.36.3-0ubuntu0.20.04.1
npm		npm			Available	6.14.4+ds-1ubuntu2
*nsis		nsis			Available	3.05-2
pkg-config	pkg-config		Installed	0.29.1

* - Optional Dependency

Diagnosis
---------
Your system has missing dependencies!

Required package(s) installation details: 
  - libgtk-3: sudo apt install libgtk-3-dev
  - libwebkit: sudo apt install libwebkit2gtk-4.0-dev
  - npm: sudo apt install npm
```

So I did a bit of experimentation:

```
ian@ubuntu-focal-qemu:~$ export LC_ALL=en_US.utf8
ian@ubuntu-focal-qemu:~$ apt list -qq libgtk-3-dev
libgtk-3-dev/focal-updates,now 3.24.20-0ubuntu1.1 amd64 [已安装]
libgtk-3-dev/focal-updates 3.24.20-0ubuntu1.1 i386
ian@ubuntu-focal-qemu:~$ export LANG=en_US.utf8
ian@ubuntu-focal-qemu:~$ apt list -qq libgtk-3-dev
libgtk-3-dev/focal-updates,now 3.24.20-0ubuntu1.1 amd64 [已安装]
libgtk-3-dev/focal-updates 3.24.20-0ubuntu1.1 i386
ian@ubuntu-focal-qemu:~$ export LANGUAGE=en_US
ian@ubuntu-focal-qemu:~$ apt list -qq libgtk-3-dev
libgtk-3-dev/focal-updates,now 3.24.20-0ubuntu1.1 amd64 [installed]
libgtk-3-dev/focal-updates 3.24.20-0ubuntu1.1 i386
```

And then opened another terminal and checked that only setting LANGUAGE did the trick, using "en" for maximum compatibility:

```
ian@ubuntu-focal-qemu:~$ apt list -qq libgtk-3-dev
libgtk-3-dev/focal-updates,now 3.24.20-0ubuntu1.1 amd64 [已安装]
libgtk-3-dev/focal-updates 3.24.20-0ubuntu1.1 i386
ian@ubuntu-focal-qemu:~$ export LANGUAGE=en
ian@ubuntu-focal-qemu:~$ apt list -qq libgtk-3-dev
libgtk-3-dev/focal-updates,now 3.24.20-0ubuntu1.1 amd64 [installed]
libgtk-3-dev/focal-updates 3.24.20-0ubuntu1.1 i386
```

So, a quick fix to that function you pointed me to, on a clean terminal:

```
ian@ubuntu-focal-qemu:~$ apt list -qq libgtk-3-dev
libgtk-3-dev/focal-updates,now 3.24.20-0ubuntu1.1 amd64 [已安装]
libgtk-3-dev/focal-updates 3.24.20-0ubuntu1.1 i386
ian@ubuntu-focal-qemu:~$ wails doctor
Wails CLI v2.0.0-beta.38

Scanning system - Please wait (this may take a long time)...Done.

System
------
OS:		Ubuntu
Version: 	20.04
ID:		ubuntu
Go Version:	go1.18.3
Platform:	linux
Architecture:	amd64

Wails
------
Version: 		v2.0.0-beta.38
Revision:		1cbc34eff8682e4daec25dd3bddb5bbc73ec09d5
Modified:		true
Package Manager: 	apt

Dependency	Package Name		Status		Version
----------	------------		------		-------
*docker		docker.io		Available	20.10.12-0ubuntu2~20.04.1
gcc		build-essential		Installed	12.8ubuntu1.1
libgtk-3	libgtk-3-dev		Installed	3.24.20-0ubuntu1.1
libwebkit	libwebkit2gtk-4.0-dev	Installed	2.36.3-0ubuntu0.20.04.1
npm		npm			Available	6.14.4+ds-1ubuntu2
*nsis		nsis			Available	3.05-2
pkg-config	pkg-config		Installed	0.29.1

* - Optional Dependency

Diagnosis
---------
Your system has missing dependencies!

Required package(s) installation details: 
  - npm: sudo apt install npm
```

Success! :tada: 

PR incoming! :smile:
